### DISCUSSION TITLE
Is this library still maintained?

### DISCUSSION THREAD

**Discussion by ruslaniv** [scikit-multilearn/scikit-multilearn#257] [General]
It looks like the last commit was 3 years ago, so I was wondering if this library is still maintained? Are there any plans for maintaining it in the future? 

I'd like to try and use it in small scale production project but if it is abandoned then I'd have to reconsider.

**Comment by Alex-Bujorianu** [comment]
I have the same question. Issue #230  remains unsolved and has to be fixed manually, which is not a sign of well maintained code.

**Comment by arav-agarwal2** [comment]
Same here - we should probably ask for someone to take over the repo if the original owners are not available.

**Comment by Alex-Bujorianu** [comment]
> Same here - we should probably ask for someone to take over the repo if the original owners are not available.

How do we do this? I’m new to Github.

**Comment by arav-agarwal2** [comment]
It looks like the original contributors are still active ( even if the Slack group is closed ).

I, or someone else, can volunteer to take over, and just start cranking things out, but we'd need one of the original contributors to give us access

**Comment by arav-agarwal2** [comment]
I'd be happy to at least help manage the current set of issues, and start pulling in requests. I've worked a bit on developing packages, and can add testing to the repo as a first way to make sure both current methods work and that later methods work better.

**Comment by niedakh** [comment]
I can add you to the repo team if you want and share the pypi credentials, we could use slack for discussions as its already set up - what do you mean that the group is closed? Would you prefer a different communication channel?

**Comment by niedakh** [comment]
I've added you to a new maintainers team, we can also do some discussions here after you accept the invitations https://github.com/orgs/scikit-multilearn/teams/maintainers/discussions

**Comment by Alex-Bujorianu** [comment]
Hi Piotr @niedakh did you also add @arav-agarwal2 to the maintainers team?

**Comment by niedakh** [comment]
> Hi Piotr @niedakh did you also add @arav-agarwal2 to the maintainers team?

Yes

**Comment by arav-agarwal2** [comment]
I'm moving the discussion about that to the discussions page, 

but when I try to join the slack I get a message that I do not have an account for that workspace.

**Comment by arav-agarwal2** [comment]
Following up here - can someone answer my notes on the discussions page?

**Comment by Alex-Bujorianu** [comment]
Hi there @arav-agarwal2 I do not see any discussions on the Github page.

**Comment by Alex-Bujorianu** [comment]
@niedakh Can you share the PyPi credentials with me and @arav-agarwal2 ? I would like to publish our changes on PyPi now that I have closed issue #224.

**Comment by Taytay** [comment]
Thanks for this library and thanks others for taking on maintenance responsibilities. I am suffering from the bug that was fixed in #224, so I'd love to see a new release!

**Comment by arav-agarwal2** [comment]
We’re hoping to get that up as soon as possible; waiting on getting pypi access to all maintainers and figuring out the backlog

**Comment by ChristianSch** [comment]
I guess this lib is still missing capable reviewers and people who have a need and the time to use this. It's also in dire need of tests, linting, formatting, modern repository standards. Github actions and maybe a bot could help with a lot of that.
